Jenkinsfile

```
@@ -47,7 +47,7 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def daos_packages_version = ""
 def el7_component_repos = ""
-def component_repos = ""
+def component_repos = "spdk@PR-11 dpdk@PR-9"
 def daos_repo = "daos@${env.BRANCH_NAME}:${env.BUILD_NUMBER}"
 def el7_daos_repos = el7_component_repos + ' ' + component_repos + ' ' + daos_repo
 def functional_rpms  = "--exclude openmpi openmpi3 hwloc ndctl spdk-tools " +
```

```suggestion
def functional_rpms  = "--exclude openmpi openmpi3 hwloc ndctl " +
```

We don't need to explicitly specify this.  The RPM will make sure it's installed.  Unless it's needed by `run_tests.sh`, in which case it should be moved there specifically.

---

IMHO, the workflow should be that the person who starts a conversation is the one who resolves it since the person who starts it needs/wants to confirm that their concern was addressed.

In any case, you are still getting errors in test because you have not removed `spdk-tools` from here as suggested above.  The version range in the daos RPMs is working, but it's this specification of `spdk-tools` that is picking up the latest one.

---

I clicked commit suggestion , but for some reason didn't seem to apply yesterday